activiti-spring-process-extensions/src/main/java/org/activiti/spring/process/variable/types/JavaObjectExtensionVariableType.java

```diff
@@ -0,0 +1,37 @@
+package org.activiti.spring.process.variable.types;
+
+import java.util.List;
+
+import org.activiti.engine.ActivitiException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JavaObjectExtensionVariableType extends ExtensionVariableType {
```

Comment:
@ryandawsonuk same here.. name too long and it is not adding any value: JavaVariableType will do, is there any reason why not to use that simple name?